lib/stats.js

```diff
@@ -99,44 +56,20 @@ module.exports.formatDataForDisplay = (inData) => {
     labels: [],
     onlineParticipantsSeries: [],
     activeConfsSeries: [],
-    USE_OVH_ROOM_API: config.USE_OVH_ROOM_API,
-  }
-  if (config.USE_OVH_ROOM_API) {
-    outData.bookedRoomsSeries = []
-  } else {
-    outData.freePhoneNumbersSeries = []
-    outData.phoneNumbersSeries = []
-    outData.bookedPhoneNumbersSeries = []
   }
+  outData.bookedRoomsSeries = []
```

Comment:
du coup on peut directement le mettre dans outData

Reply:
You're right